persistence/blockstore/block_store.go

```diff
@@ -43,6 +49,31 @@ func NewBlockStore(path string) (BlockStore, error) {
 	}, nil
 }
 
+// StoreBlock takes a coreType Block and stores it.
+func (bs *blockStore) StoreBlock(height uint64, block *coreTypes.Block) error {
+	b, err := codec.GetCodec().Marshal(block)
+	if err != nil {
+		return err
+	}
+	// TECHDEBT add a proper logger to blockstore
+	// bs.logger.Info().Uint64("height", block.BlockHeader.Height).Msg("Storing block in block store")
+	return bs.kv.Set(utils.HeightToBytes(height), b)
+}
+
+// GetBlock returns a coreTypes Block at the given height.
+func (bs *blockStore) GetBlock(height uint64) (*coreTypes.Block, error) {
+	var block coreTypes.Block
```

Comment:
NIT: Either move this above the line where we first use it or use named return types.